examples/Console/TestOtlpExporter.cs

```diff
@@ -34,10 +34,10 @@ internal static object Run(string endpoint, string protocol)
              * launch the OpenTelemetry Collector with an OTLP receiver, by running:
              *
              *  - On Unix based systems use:
-             *     docker run --rm -it -p 4317:4317 -v $(pwd):/cfg otel/opentelemetry-collector:0.33.0 --config=/cfg/otlp-collector-example/config.yaml
+             *     docker run --rm -it -p 4317:4317 -p 4318:4318 -v $(pwd):/cfg otel/opentelemetry-collector:0.48.0 --config=/cfg/otlp-collector-example/config.yaml
```

Comment:
do we need to bump version of our proto files, before moving to newer otel collector...

Reply:
Double-checking that ...

Reply:
I tested all signals and they are working so no need to change those right now. That said proto files are old and inconsistent. The inconsistency is not a problem, since different signals can use different versions. That said it makes hard to tell exactly which versions the package is using. The collector is using proto version 0.15.0 and here is a quick list from the repo files:

```terminal
logs_service.proto: a388874f Updating proto files (#1409).

metrics_service.proto: dacae6d5 Update OpenTelemetry Protocol Exporter (#643).

trace_service.proto: 9f37c7d5 Update OTLP protos to v0.9.0 (#2088).

common.proto: 436fdebb Update OTLP proto to v0.11.0 (#2606).

logs.proto: 436fdebb Update OTLP proto to v0.11.0 (#2606).

metrics_config_service.proto: 9f37c7d5 Update OTLP protos to v0.9.0 (#2088).

metrics.proto: 436fdebb Update OTLP proto to v0.11.0 (#2606).

resource.proto: c7253f59 Update to the latest stable OTLP proto files (#758).

trace.proto: 436fdebb Update OTLP proto to v0.11.0 (#2606).

trace_config.proto: a388874f Updating proto files (#1409).
```